db/migrate/20230609142126_add_user_ref_to_comment.rb

```diff
@@ -2,6 +2,6 @@
 
 class AddUserRefToComment < ActiveRecord::Migration[7.0]
   def change
-    add_reference :comments, :author_id, null: false, foreign_key: true
+    add_reference :comments, :author, column: :author_id, null: false, foreign_key: { to_table: :users }
```

Comment:
<img width="976" alt="image" src="https://github.com/mudasiri/mybloggerapp/assets/65068771/ba104beb-1db6-4700-be08-15e03320f2fc">

- [ ] Kindly make sure solve all your migration errors the same way you did in the above. check in all the migration that are referencing the user model, the should specify the table. like this

for the post table
```rb
add_reference :posts, :author, column: :author_id, null: false, foreign_key: { to_table: :users }
```

for the comment  table
```rb
add_reference :comments, :author, column: :author_id, null: false, foreign_key: { to_table: :users }
```
for the like table
```rb
add_reference :likes, :author, column: :author_id, null: false, foreign_key: { to_table: :users }
```